src/server/Document/component.jsx

```diff
@@ -69,11 +69,12 @@ const Document = ({
         )}
       </head>
       <body>
-        {/* eslint-disable react/no-danger */
-        /* disabling the rule that bans the use of dangerouslySetInnerHTML until a more appropriate implementation can be implemented */}
+        {/* disabling the rule that bans the use of dangerouslySetInnerHTML until a more appropriate implementation can be implemented */}
+        {/* eslint-disable-next-line react/no-danger */}
         <div id="root" dangerouslySetInnerHTML={{ __html: app }} />
         {scriptsAllowed && (
           <script
+            /* eslint-disable-next-line react/no-danger */
```

Comment:
Did we try @jroebu14's suggestion at https://github.com/bbc/simorgh/pull/4051/files#r331897874?

Reply:
Yes. Kindly see my comment [here](https://github.com/bbc/simorgh/issues/2622#issuecomment-539897078).